alignak_module_backend/broker/module.py

```diff
@@ -861,7 +864,7 @@ def manage_brok(self, brok):
                 logger.debug("Got contact update status brok: %s", brok.data)
                 self.update_status(brok)
 
-            if brok.type == 'new_conf':
+            if brok.type == 'new_conf' or not self.loaded:
```

Comment:
Which means that any received brok will set the `self.loaded` to True ? Is it what you expect?